cobigen-setup/files/commandsAndOptions.md

```diff
@@ -0,0 +1,38 @@
+* [generate, g]: Command used for code generation.

```

Comment:
In this step you can add a link to the already existing CobiGen Tutorial https://katacoda.com/devonfw/scenarios/cobigen
In this the generate command is used